### ISSUE TITLE
mod_signup: Schema initialization error:

### ISSUE THREAD

**Issue by fredpook** [zotonic/zotonic#1051]
A fresh install of a site crashes on ACL checks. For instance mod_signup can't startup.
Error:

```
12:11:31.847 [error] [reinwardtcommunity] Error starting module mod_signup, Schema initialization error:
{function_clause,[{lists,any,[#Fun<acl_user_groups_checks.2.77679079>,332],[{file,"lists.erl"},{line,1222}]},{acl_user_groups_checks,can_rsc_1,6,[{file,"modules/mod_acl_user_groups/support/acl_user_groups_checks.erl"},{line,344}]},{acl_user_groups_checks,default_content_group,2,[{file,"modules/mod_acl_user_groups/support/acl_user_groups_checks.erl"},{line,232}]},{acl_user_groups_checks,acl_rsc_update_check,3,[{file,"modules/mod_acl_user_groups/support/acl_user_groups_checks.erl"},{line,184}]},{lists,foldl,3,[{file,"lists.erl"},{line,1261}]},{m_rsc_update,update_transaction_fun_props,3,[{file,"src/models/m_rsc_update.erl"},{line,272}]},{z_db,'-transaction1/2-fun-0-',3,[{file,"src/db/z_db.erl"},{line,139}]},{timer,tc,2,[{file,"timer.erl"},{line,179}]}]}
12:11:31.848 [error] [reinwardtcommunity] Module mod_signup start error {error,{schema_init,{function_clause,[{lists,any,[#Fun<acl_user_groups_checks.2.77679079>,332],[{file,"lists.erl"},{line,1222}]},{acl_user_groups_checks,can_rsc_1,6,[{file,"modules/mod_acl_user_groups/support/acl_user_groups_checks.erl"},{line,344}]},{acl_user_groups_checks,default_content_group,2,[{file,"modules/mod_acl_user_groups/support/acl_user_groups_checks.erl"},{line,232}]},{acl_user_groups_checks,acl_rsc_update_check,3,[{file,"modules/mod_acl_user_groups/support/acl_user_groups_checks.erl"},{line,184}]},{lists,foldl,3,[{file,"lists.erl"},{line,1261}]},{m_rsc_update,update_transaction_fun_props,3,[{file,"src/models/m_rsc_update.erl"},{line,272}]},{z_db,'-transaction1/2-fun-0-',3,[{file,"src/db/z_db.erl"},{line,139}]},{timer,tc,2,[{file,"timer.erl"},{line,179}]}]}}} 
```

**Comment by fredpook** [comment]
Other ACL checks in templates also crash. Error
```
14:02:59.778 [error] webmachine error: path="/nl/page/335/test-text": {error,{error,function_clause,[{m_acl,m_find_value,[is_allowed,{m,m_acl,undefined},{context,reinwardtcommunity,{wm_reqdata,#Port<0.53461>,{dict,4,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[[ua_class|desktop],[ua_props,{is_user_select,true},{has_pointer,true},{is_desktop,true},{is_crawler,false},{is_wireless_device,false},{is_tablet,false},{ajax_support_javascript,true},{device_os,<<"-">>},{displayWidth,1920},{displayHeight,1200},{inputDevices,<<"-">>},{parentId,<<"generic-">>},{model,<<"browser">>},{vendor,<<"desktop">>},{id,<<"desktopDevice">>}]],[[zotonic_host|reinwardtcommunity]],[[controller_module|controller_page]],[],[],[],[],[],[],[],[],[],[]}}},undefined,"::1",undefined,{wm_log_data,887906566,undefined,{1444,132979,403371},'GET',{11,{"host",{'Host',"reinwardtcommunity.dev"},{"connection",{'Connection',"close"},{"accept",{'Accept',"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8"},nil,{"accept-encoding",{'Accept-Encoding',"gzip, deflate, sdch"},nil,{"accept-language",{'Accept-Language',"en-US,en;q=0.8,nl;q=0.6,ru;q=0.4,es;q=0.2,ar;q=0.2"},nil,nil}}},{"cookie",{'Cookie',"z_ua=c%3Ddesktop%26u%3D1%26t%3D1%26w%3D1920%26h%3D1200; z_logon=; z_sid=tSOepB3YwUGBRzLE5sUDmGgq4shQgQeq"},nil,nil}},{"x-real-ip",{"X-Real-Ip","192.168.33.1"},{"x-forwarded-for",{'X-Forwarded-For',"192.168.33.1"},{"upgrade-insecure-requests",{"Upgrade-Insecure-Requests","1"},{"referer",{'Referer',"http://reinwardtcommunity.dev/nl/admin/overview"},nil,nil},{"user-agent",{'User-Agent',"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.101 Safari/537.36"},nil,nil}},nil},nil}}},"::1","/nl/page/335/test-text",{1,0},404,0,undefined,undefined,undefined},'GET',http,{1,0},[],"/nl/page/335/test-text","/nl/page/335/test-text",{dict,6,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[zotonic_host|reinwardtcommunity],[id,51,51,53]],[[z_language,110,108]],[],[[slug,116,101,115,116,45,116,101,120,116]],[[zotonic_dispatch|page],[zotonic_dispatch_path,"page","335","test-text"]],[],[],[],[],[],[],[]}}},[],"../../..",500,1073741824,[{"z_ua","c%3Ddesktop%26u%3D1%26t%3D1%26w%3D1920%26h%3D1200"},{"z_logon",[]},{"z_sid","tSOepB3YwUGBRzLE5sUDmGgq4shQgQeq"}],[],[{"accept","text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8"},{"accept-encoding","gzip, deflate, sdch"},{"accept-language","en-US,en;q=0.8,nl;q=0.6,ru;q=0.4,es;q=0.2,ar;q=0.2"},{"connection","close"},{"cookie","z_ua=c%3Ddesktop%26u%3D1%26t%3D1%26w%3D1920%26h%3D1200; z_logon=; z_sid=tSOepB3YwUGBRzLE5sUDmGgq4shQgQeq"},{"host","reinwardtcommunity.dev"},{"referer","http://reinwardtcommunity.dev/nl/admin/overview"},{"upgrade-insecure-requests","1"},{"user-agent","Mozilla/5.0 (Windows
NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.101 Safari/537.36"},{"x-forwarded-for","192.168.33.1"},{"x-real-ip","192.168.33.1"}],not_fetched_yet,false,{4,{"cache-control",{"Cache-Control","no-store, no-cache, must-revalidate, post-check=0, pre-check=0"},nil,{"expires",{"Expires","Wed, 10 Dec 2008 14:30:00 GMT"},{"content-type",{"Content-Type","text/html; charset=utf-8"},nil,nil},{"p3p",{"P3P","CP=\"NOI ADM DEV PSAi COM NAV OUR OTRo STP IND DEM\""},nil,nil}}}},"identity",undefined,"text/html","utf-8",<<>>,true,[],none,[{last_modified,undefined},{generate_etag,undefined},{content_encodings_provided,["identity"]},{charsets_provided,[{"utf-8",#Fun<controller_page.0.17637224>}]},{content_types_provided,[{"text/html",to_html}]}]},controller_page,<0.2535.0>,<<"tSOepB3YwUGBRzLE5sUDmGgq4shQgQeq">>,<0.11711.0>,<<"DtrqfuO8WZxPYOzaSj1u">>,undefined,'z_depcache$reinwardtcommunity','z_notifier$reinwardtcommunity','z_session_manager$reinwardtcommunity','z_dispatcher$reinwardtcommunity','z_template$reinwardtcommunity','z_scomp$reinwardtcommunity','z_dropbox$reinwardtcommunity','z_pivot_rsc$reinwardtcommunity','z_module_indexer$reinwardtcommunity',303221,undefined,{'z_db_pool$reinwardtcommunity',z_db_pgsql},...}],...},...]}}
```

**Comment by mworrell** [comment]
What is the list of modules being installed?

**Comment by fredpook** [comment]
mod_base,
   mod_menu,
   mod_oauth,
   mod_search,
   mod_oembed,
   mod_atom_feed,
   mod_translation,
   mod_signal,
   mod_logging,

   mod_seo,
   mod_seo_google,
   mod_seo_sitemap,

   mod_authentication,
   mod_acl_adminonly,
   mod_acl_user_groups,
   mod_content_groups,
   mod_signup,
   mod_editor_tinymce,
   mod_mqtt,

   mod_admin,
   mod_admin_category,
   mod_admin_config,
   mod_admin_identity,
   mod_admin_modules,
   mod_admin_predicate,

   mod_l10n,
   mod_rest,
   mod_backup,

   mod_artwork,
   mod_bootstrap,

   mod_ginger_default,
   mod_ginger_admin,
   mod_ginger_search,
   mod_ginger_auth,
   mod_ginger_edit,

   mod_video,
   mod_video_embed,

   mod_import_anymeta_dispatch,
   mod_import_anymeta,
   mod_import_utils,
   mod_comment,
   mod_facebook,
   mod_twitter

**Comment by Dirklectisch** [comment]
This error is also preventing us from editing person pages from /admin.

**Comment by mworrell** [comment]
This is fixed in https://github.com/zotonic/zotonic/commit/a8e206fb89bf04af777cffb42660ff3b13545017

This fix was already done on master but not merged back into the 0.13.x
